src/main/java/com/cleanroommc/modularui/api/drawable/IDrawable.java

```diff
@@ -175,5 +178,15 @@ public DrawableWidget(IDrawable drawable) {
         public void draw(ModularGuiContext context, WidgetTheme widgetTheme) {
             this.drawable.drawAtZero(context, getArea(), widgetTheme);
         }
+
+        @Override
+        public boolean equals(Object obj) {
+            if (obj instanceof IDrawable drawable)
+                return this.drawable == drawable;
+            else if (obj instanceof DrawableWidget drawableWidget)
+                return this.drawable == drawableWidget.drawable;
+
+            return false;
+        }
```

Comment:
???

Reply:
i figured it could be nice to be able to compare drawable widgets in certain cases, but i don't really care enough to keep it if it isn't desirable